frontend/src/atoms.ts

```diff
@@ -187,3 +187,13 @@ export function useSearchAutocompleteLimit() {
   const settings = useRecoilValue(settingsState)
   return useMemo(() => parseInt(settings.SEARCH_AUTOCOMPLETE_LIMIT ?? '10000'), [settings])
 }
+
+export function useIsGlobalHub() {
+  const crds = useRecoilValue(customResourceDefinitionsState)
+  return useMemo(
+    () =>
+      crds.filter((crd) => crd.metadata.name === 'multiclusterglobalhubs.operator.open-cluster-management.io').length >
```

Comment:
- Do we really need to fetch all CRDs? It would be more efficient to filter on the server side.
- `crds.find(...)` is better because it stops the loop once it finds a match.